design/baremetal-operator/detached-annotation.md

```diff
@@ -78,8 +78,8 @@ baremetalhost.metal3.io/detached: ""
 The value is ignored, similar to the `paused` annotation and could optionally
 include context from the system/user which applies the annotation.
 
-This annotation will only be consumed while the BMH is in either `Provisioned`
-or `ExternallyProvisioned` state, in all other cases it is ignored.
+This annotation will only be consumed while the BMH is in either `Provisioned`,
+`ExternallyProvisioned` or `Ready`/`Available` state, in all other cases it is ignored.
```

Comment:
`Available` state is not implemented yet, no?

Reply:
This is based on this [comment](https://github.com/metal3-io/baremetal-operator/pull/967#discussion_r703677051)